web-extension/background-engine.js

```diff
@@ -17,19 +55,28 @@ bmcMessaging.addWindowHandler(
     evData => evData.type === 'computation'
               && evData.module === 'sources'
               && evData.computation === 'URL:Generate:Request',
-    evData => { // Don't receive `sender`, as we don't use it.
+    (evData, sender, sendResponse) => { // Don't receive `sender`, as we don't use it.
         LOGS.log('S68', {'evData' :evData});
-        const answerEv = {
-            type: 'computation',
-            module: 'sources',
-            computation: 'URL:Generate:Response',
-            resource: {
-                reader: evData.resource.reader,
-                comic: evData.resource.comic,
-                url: bmcSources.computeURL(evData.resource.reader,
-                                           evData.resource.comic),
+        bmcData.getComic(evData.resource.id, (err, comic) => {
+            let answerEv = {
+                type: 'computation',
+                module: 'sources',
+                computation: 'URL:Generate:Response',
+                resource: {
+                    err: false,
```

Comment:
Why providing the fields if they are at `false` and `null`?

Reply:
simply setting the common object's structure, for consistency.
Is that a problem for you ?

Reply:
No, just wondered.